scripts/xtensa-build-zephyr.py

```diff
@@ -391,12 +393,14 @@ def build_platforms():
 		if args.cmake_args:
 			build_cmd += args.cmake_args
 
-		overlays = []
+		overlays = [str(item.resolve(True)) for item in args.overlay]
```

Comment:
So much done in a single line and still very readable... impressive!

Reply:
tested with `./scripts/xtensa-build-zephyr.py tgl -o overlays/tgl/ipc4_overlay.conf overlays/common/debug_overlay.conf`